ci-operator/step-registry/upi/install/powervs/cluster/upi-install-powervs-cluster-ref.yaml

```diff
@@ -0,0 +1,34 @@
+ref:
+  as: upi-install-powervs-cluster
+  from: upi-installer
+  commands: upi-install-powervs-cluster-commands.sh
+  grace_period: 10m
+  resources:
+    requests:
+      cpu: 10m
+      memory: 100Mi
+  dependencies:
+  - name: "release:latest"
+    env: OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE
+  env:
+  - name: RHCOS_IMAGE_FILENAME
+    default: "rhcos-415-92-202310161140-0-ppc64le-powervs.ova.gz"
```

Comment:
We should pull this out of the openshift-installer so we get the latest every time.

Reply:
```
# rhcos_import_image_filename        = "rhcos-414-92-202307050443-0-ppc64le-powervs.ova.gz"
COREOS_URL=$(openshift-install coreos print-stream-json | jq -r '.architectures.ppc64le.artifacts.powervs.formats."ova.gz".disk.location')
COREOS_FILE=$(echo ${COREOS_URL} | sed 's|/| |g' | awk '{print $NF}')
COREOS_NAME=$(echo ${COREOS_FILE} | sed 's|\.ova\.gz||' | tr '.' '-' | sed 's|-0-powervs-ppc64le||g')
```